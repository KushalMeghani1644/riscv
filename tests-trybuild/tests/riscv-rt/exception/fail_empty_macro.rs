#[riscv_rt::exception]
fn my_exception() {}

fn main() {}
