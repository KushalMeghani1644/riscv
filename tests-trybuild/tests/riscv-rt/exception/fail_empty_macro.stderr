error: `#[exception]` attribute expects a path to a variant of an enum that implements the riscv_rt :: ExceptionNumber trait.
 --> tests/riscv-rt/exception/fail_empty_macro.rs:1:1
  |
1 | #[riscv_rt::exception]
  | ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `riscv_rt::exception` (in Nightly builds, run with -Z macro-backtrace for more info)
